### DISCUSSION TITLE
Setting up WAVE SDK development environment

### DISCUSSION THREAD

**Discussion by Nayananga** [h2oai/wave#1105] [Q&A] [answered]
Hi, I was trying to set up the WAVE SDK development environment locally in my 2019 MBP, OSx version 11.4 (20F71), And I encountered an error. Could you please tell me what did I do wrong? 

here is the log
[2021-11-03T11_06_07_330Z-debug.log](https://github.com/h2oai/wave/files/7467177/2021-11-03T11_06_07_330Z-debug.log)

**Comment by mturoci** [comment, marked as answer]
You are using just a week old Node version, seems like `node-sass` is not catching up. Try with these versions instead:
![image](https://user-images.githubusercontent.com/64769322/140052965-623e9d94-d164-4ad2-9057-1a202828d94a.png)

**Reply by Nayananga** [reply]
Hi, Setting the node version to v12.22.7 solved the issue. Thanks
